config/src/test/java/org/axonframework/config/EventProcessingModuleTest.java

```diff
@@ -369,6 +371,23 @@ public void testConfigureErrorHandlerPerEventProcessor() throws Exception {
         }
     }
 
+    @Test
+    public void testShutdown() throws NoSuchFieldException {
+        Configuration configuration = configurer
+                .configureEmbeddedEventStore(c -> new InMemoryEventStorageEngine())
+                .eventProcessing(ep -> ep.registerEventHandler(c -> new SubscribingEventHandler())
+                        .registerEventHandler(c -> new TrackingEventHandler())
+                        .registerTrackingEventProcessor("tracking")
+                        .registerSubscribingEventProcessor("subscribing"))
+                .start();
+
+        configuration.shutdown();
+
+        assertNull(getFieldValue(SubscribingEventProcessor.class.getDeclaredField("eventBusRegistration"),
+                        configuration.eventProcessingConfiguration().eventProcessor("subscribing", SubscribingEventProcessor.class).get()));
+        assertFalse(configuration.eventProcessingConfiguration().eventProcessor("tracking", TrackingEventProcessor.class).get().isRunning());
```

Comment:
What is the actual reason why you didn't use https://riptutorial.com/mockito/example/18318/verify-order-of-calls ? It would confirm that order is OK and using the reflection won't be necessary. What do you think?

Reply:
I did not think of using mocks.  I will change this if the PR gets approved